src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -217,21 +217,21 @@ private static ParserInstance getParser() {
 	 */
 	private static final FileFilter scriptFilter =
 		f -> f != null
-			&& (f.isDirectory() || StringUtils.endsWithIgnoreCase(f.getName(), ".sk"))
-			&& !f.getName().startsWith("-");
-	
+			&& (f.isDirectory() && !f.getName().startsWith(".") || !f.isDirectory() && StringUtils.endsWithIgnoreCase("" + f.getName(), ".sk"))
```

Comment:
Why the `"" + ` here?

Reply:
It was probably left over that I forgot about, should be good now :+1: